woocommerce/woocommerce
# Couple of caching fixes for daily tests

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



This PR aims to address a couple of instability issues in the API tests running on the daily external host.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Run the API tests locally using `pnpm env:start && pnpm test:api-pw`
2. Create an .env file containing the `API_BASE_URL`, `USER_KEY` and `USER_SECRET` for the WC Core Nightly API Smoke test site.  Run the tests again using `pnpm test:api-pw`
3. Ensure that CI checks pass including a daily run based on this branch.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
